cd-cli/cnf-templates/complete-pipeline.yaml

```diff
@@ -5053,7 +5053,7 @@ Resources:
     Type: AWS::Events::Rule
     Properties:
       Description: "e2eScheduledRule"
-      ScheduleExpression: "cron(0 7-19/2 ? * MON-FRI *)"
```

Comment:
@EmanueleBonofiglio lto fe abbiamo messo "cron(0 23 ? * MON-FRI *)" dobbiamo usare la tua stessa regola?

Reply:
Non penso che sia necessario usare la stessa regola, anzi se sia lato FE che BE si fà la scelta di una singola build notturna tanto vale schedularle in orari diversi, idealmente in maniera che non siano sovrapposti, così da avere esecuzioni "pulite" senza rischi di conflitti (so che allo stato attuale sono improbabili se non impossibili, ma dato che ci siamo)

Reply:
va bene! quindi possiamo mettere sul cron di fe la regola  "cron(3 4 ? * MON-FRI *)" o lasciamo quella attuale?

Reply:
Si se preferite va benissimo.